modules/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpConnectionManager.java

```diff
@@ -331,4 +332,13 @@ private void populateSenderConfigurationOptions(SenderConfiguration senderConfig
         }
         senderConfiguration.setSocketIdleTimeout((int) endpointTimeout);
     }
+
```

Comment:
Don't we already have a util method for this? .. how do we do this in other places ..

Reply:
It's in the launcher.. I don't think there are other places which requires this. In other places it's usually a constant string which needs to be written to the console. A simple solution for this would be to add some prefix to the message?

Reply:
Yeah, in that case, I'm just thinking, isn't the launcher that is suppose to log a message like this. Which I guess happens when we throw the exception from here. This seems to be like we are bypassing the launcher and printing our own formatted message to the standard error. @sameera-jayasoma any suggestions here? .. maybe we need some type of a central logging API from the ballerina-lang so any component can log to stdout/stderror in an unified manner.

Reply:
Yes, it is.. But the launcher only prints such messages when catching an exception. In this case, we need to be able to print out the cause for the connector startup to fail for each of the server connectors. But we don't want to terminate the runtime if there is at least one server connector which started properly. Another place where such messages are printed is in `HttpConnectorPortBindingListener`. 

+1 for logging from a central place. We can add it to the `ballerina-logging` module. We can then avoid direct use of `System.out/err` uses altogether across all the modules.

Reply:
I see, for now this is okay then. Please make a note that we have to do this possible improvement later.